content/guides/basic-setup/index.md

```diff
@@ -13,48 +13,55 @@ If you require further help, please visit our forums or ask in our Discord's hel
 
 ## How to dump your PS2 BIOS
 
-Dumping your PS2 BIOS requires a mod to your PS2 to allow arbitrary code execution.
+Dumping your PS2 BIOS is conceptually a two-step process:
+1. "Hack" your PS2 to allow arbitrary code execution
```

Comment:
1. Calling it hack doesn't sit right with me. It's calling softmodding or hardmodding your console
2. The goal is to make this guide easy for beginners to understand. Warning, this is my opinion: Using terms like 'arbitrary code execution' can seem intimidating and increase the likelihood of a user saying "screw it this is too hard"

Reply:
Changed to "modify PS2 operation" in upcoming commit (:crossed_fingers: ).